README.md

```diff
@@ -700,6 +702,44 @@ In this example, all saves to `MyTable` will raise an `Dynamoid::Errors::StaleOb
 
 Calls to `update` and `update!` also increment the `lock_version`, however they do not check the existing value. This guarantees that a update operation will raise an exception in a concurrent save operation, however a save operation will never cause an update to fail. Thus, `update` is useful & safe only for doing atomic operations (e.g. increment a value, add/remove from a set, etc), but should not be used in a read-modify-write pattern.
 
+
+### Backoff strategies
+
+
+You can use several methods what do efficiently in batch mode like `.find_all` and `.import`.
```

Comment:
This sentence is not clear to me.  Can you clarify the meaning abit around "what do efficiently"?

Reply:
@pboling They use batch operations like `batch_get_item` and `batch_write_item`. So they make fewer HTTP requests.

Reply:
Maybe like "You can use several methods that run efficiently in batch mode like `.find_all`..."

It's unclear to me as well.

Reply:
I think it is a translation error.  If you can try restating it with other words I might be able to understand it.

Perhaps:

> There are some built-in exponential backoff strategy configurations that allow methods like `.find_all` and `.import` to process all items, automatically handling API batch limitations, and reprocessing failed items until they pass.

However, now that I have stated it like that... is it possible that this exponential backoff could encounter an item that will always fail to process, and thus the backoff will be in a retry loop forever?  Does the backoff have an escape hatch after a certain number of tries?